tests/workbenches/conftest.py

```diff
@@ -62,6 +76,17 @@ def default_notebook(
     # Set the correct username
     username = get_username(dyn_client=admin_client)
 
+    # Set the image path based on internal image registry status
+    try:
+        internal_image_registry.wait_for_condition(
```

Comment:
should be under `internal_image_registry`; if the resource is not ready, there's no point in continuing the flow

Reply:
done